### ISSUE TITLE
Request: Xenoblade 3 v2.1.0 update

### ISSUE THREAD

**Issue by macsimbelous** [theboy181/switch-ptchtxt-mods#31]
Please update the graphics cheats for version 2.1.0

**Comment by TobiasEl** [comment]
Yes, plis. We need it. Xenoblade 3 v 2.1.0 looks awful, we need the mods for YUZU.
BTW thank you so much for your work.

**Comment by YingDoge** [comment]
The Newest Update is Xenoblade Chronicles 3 NSP Update (v2.1.1). So Update to this version please that

**Comment by TobiasEl** [comment]
But there is no mods for version 2.1.1 (or 2.1.0):
![imagen](https://github.com/theboy181/switch-ptchtxt-mods/assets/36697892/9e7a47d4-b8d0-416f-9cca-417a42335970)

**Comment by YingDoge** [comment]
@macsimbelous 
<img width="689" alt="image" src="https://github.com/theboy181/switch-ptchtxt-mods/assets/135021273/b39d3801-abe1-4dc9-af51-644e533c989e">
You woudl want to wait for 2.20 which woudl be the final update

**Comment by YingDoge** [comment]
> But there is no mods for version 2.1.1 (or 2.1.0): ![imagen](https://user-images.githubusercontent.com/36697892/270168419-9e7a47d4-b8d0-416f-9cca-417a42335970.png)

Fair Fair My fault sorry
